ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -64,7 +64,7 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y git python
 RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y python3-pip
 @# colcon-core.package_identification.python needs at least setuptools 30.3.0
 @# pytest-rerunfailures enables usage of --retest-until-pass
-RUN pip3 install -U setuptools pytest-rerunfailures
+RUN pip3 install -U setuptools==59.6.0 pytest-rerunfailures
```

Comment:
Will there ever be patch releases of 59.x? Would it be better to go with `setuptools<60`?

Reply:
I don't have a strong opinion either - just thought I'd mention it.